driver/src/main/java/com/microsoft/playwright/impl/Driver.java

```diff
@@ -65,6 +65,25 @@ public Path driverPath() {
     return driverDir().resolve(cliFileName);
   }
 
+  public static void setRequiredEnvironmentVariables(ProcessBuilder pb) {
+    if (!pb.environment().containsKey("PW_CLI_TARGET_LANG")) {
+      pb.environment().put("PW_CLI_TARGET_LANG", "java");
```

Comment:
looks like PW_CLI_TARGET_LANG /PW_CLI_TARGET_LANG_VERSION are used beyond just codegen, let's rename  them (in separate pr) to PW_LANG / PW_LANG_VERSION before it's too late?

Reply:
Let's do this in a follow-up and on tip-of-tree only.

I filed an issue for myself regarding this: https://github.com/microsoft/playwright/issues/11450